bin/gce-create-instance

```diff
@@ -3,11 +3,12 @@
 SCOPES=${SCOPES:-storage-ro}
 
 instname=$1
+bucketname=$2
```

Comment:
This requires users to specify an additional field for bucket name, but it's not really necessary.  It's possible to store the bucketname in an environment variable set in `gcerc`.